content/docs/gke/monitoring.md

```diff
@@ -4,162 +4,7 @@ description = "Logging and monitoring for Kubeflow"
 weight = 110
 +++
 
-This guide has information about how to set up logging and monitoring for your
-Kubeflow deployment.
 
-# Logging
+[Prometheus](https://prometheus.io/) is a monitoring tool often used with Kubernetes. If you configure Kubernetes Engine Monitoring and include Prometheus support, then the metrics that are generated by services using the [Prometheus exposition format](https://github.com/prometheus/docs/blob/master/content/docs/instrumenting/exposition_formats.md) can be exported from the cluster and made visible as [external metrics](https://cloud.google.com/monitoring/api/metrics_other#externalgoogleapiscom) in Cloud Monitoring.
```

Comment:
For the Prometheus exposition format, I think it's best to use the website link rather than the GitHub link for the docs:
https://prometheus.io/docs/instrumenting/exposition_formats/